src/status_im/contexts/wallet/send/input_amount/view.cljs

```diff
@@ -334,7 +336,7 @@
              :fees            fee-formatted
              :amount          amount-text
              :receiver        (address/get-shortened-key to-address)}])
-         (when (or no-routes-found? limit-insufficient?)
+         (when no-routes-found?
```

Comment:
this component and it's bindings is becoming larger and larger. Do we think we can componentize some smaller parts of this?

e.g we make a [no-routes ...] component in this view just to make this block slightly smaller? 🤔 🙏